src/core/server/metrics/types.ts

```diff
@@ -20,14 +20,15 @@
 import { Observable } from 'rxjs';
 import { OpsProcessMetrics, OpsOsMetrics, OpsServerMetrics } from './collectors';
 
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface MetricsServiceSetup {}
 /**
  * APIs to retrieves metrics gathered and exposed by the core platform.
  *
  * @public
  */
-export interface MetricsServiceStart {
+export interface MetricsServiceSetup {
```

Comment:
In order to use it in HTTP routes in Core, this was much simpler. It could also be added tot he requesthandlercontext, but I felt this would pollute that public API a bit since this is only needed by 2 endpoints right now.